web/src/components/storage/VolumeForm.test.jsx

```diff
@@ -95,12 +95,12 @@ beforeEach(() => {
 });
 
 it("renders a control for displaying/selecting the file system type", async () => {
-  const { user } = plainRender(<VolumeForm {...props} />);
+  // use home which is not snapshotted
```

Comment:
CSpell so picky here. Maybe we can go for 

```suggestion
  // use home which is not using snapshots
```